src/Config.js

```diff
@@ -269,6 +272,22 @@ export class Config {
     }
   }
 
+  static validateGraphqlConfig(graphqlConfig: GraphqlConfig) {
+    if (!graphqlConfig) return;
+    if (Object.prototype.toString.call(graphqlConfig) !== '[object Object]') {
+      throw 'Parse Server option graphqlConfig must be an object.';
```

Comment:
Here codecov report that errors are not covered, I'll suggest covering errors, it could help developers to debug a wrong graphql config.

It's not a request change, feel free to cover errors @tstachl  

@mtrezza  what do you think ?

Reply:
otherwise LGTM

Reply:
Yes, should be an easy one, just copy/paste another test.